speasy/webservices/cda/_inventory_builder/__init__.py

```diff
@@ -28,14 +28,16 @@ def _clean_master_cdf_folder():
 
 def _download_and_extract_master_cdf(masters_url: str):
     with NamedTemporaryFile('wb') as master_archive:
-        master_archive.write(requests.get(masters_url).content)
+        r = http.get(masters_url)
+        master_archive.write(r.content)
```

Comment:
Why? The whole point here was to reduce as much a possible the scope/lifetime of r.